src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/handler/AmendApplicationAboutToSubmitHandler.java

```diff
@@ -56,7 +62,47 @@ public GenericAboutToStartOrSubmitCallbackResponse<FinremCaseData> handle(Finrem
         updateApplicantOrSolicitorContactDetails(caseData);
         updateLatestConsentOrder(callbackRequest);
 
-        return GenericAboutToStartOrSubmitCallbackResponse.<FinremCaseData>builder().data(caseData).build();
+        return GenericAboutToStartOrSubmitCallbackResponse.<FinremCaseData>builder().data(caseData).errors(errors).build();
+    }
+
+    private void checkApplicantPostCodeDetails(FinremCaseData caseData, List<String> errors) {
+        String postCode = null;
+
+        if (caseData.isApplicantRepresentedByASolicitor()) {
+            postCode = caseData.getApplicantSolicitorPostcode();
+        } else {
+            Address applicantAddress = caseData.getContactDetailsWrapper() != null
```

Comment:
Seems getContactDetailsWrapper already implemented the check null logic.

Reply:
@so99y Ah, interesting.  @hamzahtahirhmcts Try just keeping the null check on address then?